internal/providers/terraform/aws/registry.go

```diff
@@ -180,6 +181,12 @@ var FreeResources []string = []string{
 	"aws_s3_bucket_policy",
 	"aws_s3_bucket_public_access_block",
 
+	// AWS KMS
```

Comment:
This group should probably go after `// AWS IAM aws_iam_* resources` as we try to keep these grouped alphabetically - both at the group level and at the line-item level, which you've already done :)